src/app/fdctl/run/topos/fd_firedancer.c

```diff
@@ -405,6 +405,7 @@ fd_topo_firedancer( config_t * _config ) {
 
     } else if( FD_UNLIKELY( !strcmp( tile->name, "dedup" ) ) ) {
       tile->dedup.tcache_depth = config->tiles.dedup.signature_cache_size;
+      FD_TEST( fd_rng_secure( &tile->dedup.hashmap_seed, 8U ) );
```

Comment:
Do this in privileged_init of the tile, not here

Reply:
fixed